recipe/meta.yaml

```diff
@@ -20,6 +20,61 @@ source:
 build:
   number: 0
   skip: True               # [py<37]
+  entry_points:
```

Comment:
Usually, we add only `console_scripts`. I'm not 100% sure if we want all other types of entry_points

Reply:
Thanks, indeed that's not needed. I removed them.